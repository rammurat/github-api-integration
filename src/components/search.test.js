import React from 'react'
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux'

import Search from './search'

const store = {
    subscribe: () => { },
    dispatch: () => { },
    getState: () => ({})
}
// @ts-ignore
it('renders correctly', () => {
    const tree = renderer
        // @ts-ignore
        .create(<Provider store={store}><Search /></Provider>)
        .toJSON();
    // @ts-ignore
    expect(tree).toMatchSnapshot();
});