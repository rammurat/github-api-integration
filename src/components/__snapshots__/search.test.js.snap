// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa fmcpco"
>
  <form
    className="form-inline"
    onSubmit={[Function]}
  >
    <div
      className="form-group mx-sm-3 mb-2"
    >
      <label
        className="sr-only"
      >
        Password
      </label>
      <input
        className="form-control"
        id="search"
        name="search"
        onChange={[Function]}
        placeholder="JavaScript"
        type="text"
        value=""
      />
    </div>
    <button
      className="btn btn-primary mb-2"
      type="submit"
    >
      Search
    </button>
  </form>
</div>
`;
